src/components/DataView/__tests__/DataView.test.js

```diff
@@ -629,4 +629,37 @@ describe('DataView with data', () => {
       expect(row.dataset.itemId).toBe(String(data[rowIDX][headers[0]]))
     )
   })
+
+  it('should render icon if available', () => {
+    const onItemClickHandler = jest.fn()
+
+    const dataWithIcon = [
+      {
+        'Openbare Naam': 'hoofdcategorie - openbare naame',
+        fk: '177',
+        Hoofdcategorie: 'hoofdcategorie test',
+        Icoon:
+          'https://siaweuaaks.blob.core.windows.net/files/icons/categories/0-hoofdcategorie-test/glas-icon.svg?se=2023-04-25T15%3A16%3A27Z&sp=r&sv=2021-08-06&sr=b&sig=GtCGkYzJhlkzlRrVAShohBuCQ0mq%2BojItkjJvPRWFBY%3D',
+        id: 'https://acc.api.meldingen.amsterdam.nl/signals/v1/public/terms/categories/0-hoofdcategorie-test',
+      },
+      {
+        'Openbare Naam': 'Afval',
+        fk: '156',
+        Hoofdcategorie: 'Afval',
```

Comment:
bit strange these keys have spaces and capitals

Reply:
Agree. I was confused at first as well. But this is the way it works ...